text/src/autogluon/text/automm/optimization/utils.py

```diff
@@ -346,6 +369,7 @@ def apply_layerwise_lr_decay(
         lr: float,
         lr_decay: float,
         weight_decay: float,
+        efficient_finetune: Optional[str] = None,
```

Comment:
Do we also need to add this option for `apply_single_lr` and `apply_two_stages_lr`? Do the papers use layer-wise learning rate decay in efficient fine-tuning?

Reply:
No. The original paper hasn't combined bit_fit with layerwise lr decay. Also, should we remove apply_single_lr in the future?

Reply:
Since the original paper didn't use layer-wise lr decay, we may need to test whether layer-wise lr decay is a better choice than single lr for bit_fit.

Reply:
After additional experiments, I noticed that lr_decay=1.0 gives the best result. Thus, we may stick to lr_decay=1.0 when applying efficient_finetune. In addition, we may only implement that for `apply_layerwise_lr_decay` because `lr_decay=1.0` is equivalent to `apply_single_lr`.

Reply:
We can remove `apply_single_lr` later. If a user wants to use efficient fine-tuning, the user needs to customize both `efficient_finetune ` and `lr_decay `. We may need a tutorial for configuring efficient fine-tuning.